lib/index.js

```diff
@@ -112,11 +112,14 @@ const os = require('os'),
                 $self: [selfModelName, selfId]
             },
             cursor: data.cursor,
-            context: {
-                ref: data.cursor.ref,
-                httpRequestId: data.cursor.httpRequestId,
-                scriptId: data.cursor.scriptId,
-                execution: data.cursor.execution
+            args: data.cursor && {
```

Comment:
@codenirvana can it be ensured that undefined valued are cleaned up from the `args?

Reply:
Yes